packages/main/src/i18n/messagebundle.properties

```diff
@@ -91,6 +91,27 @@ COLOR_PALETTE_DIALOG_TITLE=Change Color
 #XTIT: Color Palette dialog title of the Color Picker
 COLOR_PALETTE_MORE_COLORS_TEXT=More Colors...
 
+#XACT: Aria information for the ColorPicker Alpha slider
+COLORPICKER_ALPHA_SLIDER=Alpha control
```

Comment:
why not 'Alpha slider'?
'Hue slider'

Also 'Red input' sounds better, not sure if the input itself has more labels, so that the reader reads the combination as 'edit red'/ 'red input' or similar....

Reply:
Unfortunately I couldn't find those texts specified somewhere. I've used the same texts as in the other project for the color picker sliders.
The announcement would be "Alpha control slider handle".

For the input controls you're right. We have additional announcements from the input components:
"Red edit",  "Alpha edit" etc.

Reply:
ok